recipe/meta.yaml

```diff
@@ -62,23 +83,31 @@ outputs:
         - sktime.utils
 
   - name: sktime-all-extras
+    build:
+      #FIXME: Extras environment not resolvable on aarch. Needs someone with
+      #arm machine to tests.
+      skip: true  # [aarch64]
     requirements:
+      host:
+        - python
       run:
         - python
-        - {{ pin_subpackage("sktime", max_pin="x.x.x.x") }}
-        - matplotlib-base >=3.3.2
-        - pmdarima >=1.8.0,!=1.8.1  # [not win and py<=38]
-        - scikit-posthocs >=0.6.5
-        - seaborn >=0.11.0
-        - tsfresh >=0.17.0
-        - numba >=0.53
-        # - catch22 >=0.1.0
-        - stumpy >=1.5.1
-        - tbats >=1.1.0
-        - prophet >=1.1  # [not aarch64 and not linux and py>37]
-        - filterpy >=1.4.5
-        - pykalman >=0.9.5
+        - {{ pin_subpackage("sktime", exact=True) }}
+        - matplotlib-base
+        - pmdarima
+        # NOTE: Selector based on prophet=1.0 availablility circa 9/2022
+        - prophet   # [py>=37 and py<310 and (linux64 or osx64 or win64)]
+        - pykalman
+        - scikit-posthocs
+        - seaborn
+        - stumpy
+        - tbats
+        - tsfresh
```

Comment:
I think all packages in `run_constrained` should be in the `sktime-all-extras` package as there's no mechanism to actually install `run_constrained` deps, or rather, creating the separate `sktime-all-extras` package ***is*** the way to install all optional deps.

Reply:
I agree @dhirschfeld, `sktime-all-extras` should have all deps considered required since that's the whole suite.

Reply:
See [my comment below](https://github.com/conda-forge/sktime-feedstock/pull/52#discussion_r961822155) for why I added unconstrained packages to the run_constrained section.

I didn't add any new packages to the `requirements/run` section of the `all-extras` package because I don't want to break compatibility with the previous build, and as I mentioned in the PR description, I don't know about this software to determine why so many of the extras are missing from that package. If it was up to me, I would convert this recipe to noarch and include all of the packages in `all-extras`. However, that would probably lead to broken installs. I still encourage the maintainers to convert to noarch and list all the packages for the next release 0.13.0, as mentioned in the PR description.